package.json

```diff
@@ -2,9 +2,10 @@
   "name": "aqts-capture-ecosystem-switch",
   "description": "Deployment via serverless",
   "dependencies": {
-    "serverless": "1.77.1",
+    "serverless": "2.1.1",
```

Comment:
I think this is up to `2.8.0` now, they iterate pretty quickly.

Reply:
good grief.  Not sure I want to be on the bleeding edge, but I guess if I see weird errors I can back off.